<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 100, 139);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="B_exit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="eightB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneB">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>319</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_name">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
font: 63 8pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Please enter amount of money to deposit:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sixB">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoB">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_balance">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 100, 139);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CURRENT BALANCE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_amount_withdraw">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="li_balance">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fourB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_ok">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
background-color: rgb(50, 173, 58);

font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B_back">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 119);
color: rgb(54, 100, 139);
font: 63 14pt &quot;Montserrat SemiBold&quot;;
border-radius: 20;
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 240, 245);
color: rgb(34, 64, 89);
font: 63 14pt ;Montserrat SemiBold;
border-radius: 20 ;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_error">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
